packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -2037,7 +2032,7 @@ export class ContainerRuntime
 		// Anything else is too risky, and requires ability to initialize ID compressor synchronously!
 		if (schema.runtime.idCompressorMode !== undefined) {
 			// eslint-disable-next-line @typescript-eslint/no-floating-promises
-			this.loadIdCompressor();
+			this.initIdCompressor();
```

Comment:
The 'loadIdCompressor' method invokes the 'createIdCompressorFn()' callback.

Because this no method no longer performs a dynamic load of the 'id-compressor' package, I've renamed it 'loadIdCompressor' -> 'initIdCompressor'.